File: ConsoleAppCs/PersonModel.cs

namespace ConsoleAppCs
{
    public class PersonModel
    {
        public string firstName;
        public string lastName;
        public string emailAddress;

        public string FirstName2 { get; set; }
        public string LastName2 { get; set; }
        public string EmailAddress2 { get; set; }
        public bool HasBeenGreeted { get; set; }

        public static void GreetPerson(PersonModel person)
        {
            Console.WriteLine($"Hello {person.FirstName2} {person.LastName2}");
            person.HasBeenGreeted = true;
        }


    }
}

File: ConsoleAppVb/PersonModel.vb
Namespace ConsoleAppCs
    Public Class PersonModel
        Public firstName As String
        Public lastName As String
        Public emailAddress As String

        Public Property FirstName2 As String
        Public Property LastName2 As String
        Public Property EmailAddress2 As String
        Public Property HasBeenGreeted As Boolean

        Public Shared Sub GreetPerson(person As PersonModel)
            Console.WriteLine($"Hello {person.FirstName2} {person.LastName2}")
            person.HasBeenGreeted = True
        End Sub
    End Class
End Namespace

